azkaban-common/build.gradle

```diff
@@ -16,6 +16,7 @@ model {
 
 dependencies {
   compile('com.google.guava:guava:13.0.1')
+  compile('com.h2database:h2:1.4.184')
```

Comment:
Latest is 1.4.192 in mavenCentral.
https://mvnrepository.com/artifact/com.h2database/h2/1.4.192